happy/HappyStateDelete.py

```diff
@@ -193,9 +195,38 @@ def __cleanup_host(self):
         self.__delete_host_bridges()
         self.__delete_host_links()
 
+    def __delete_internet(self):
+        """
+        delete internet isp interface
+        functionality similar to command: happy-internet -d --node onhub --interface em1 --isp em --seed 249
```

Comment:
I'd describe this as "happy-internet -d ...", avoiding the specific concrete parameter examples.

Reply:
fixed this one